src/bundle/Resources/public/js/scripts/admin.notifications.filters.js

```diff
@@ -0,0 +1,134 @@
+    const applyFiltersBtn = filtersContainerNode.querySelector('.ibexa-btn--apply');
+    const clearFiltersBtn = filtersContainerNode.querySelector('.ibexa-btn--clear');
+    const statusFilterNode = filtersContainerNode.querySelector('.ibexa-list-filters__item--statuses');
+    const typeFilterNode = filtersContainerNode.querySelector('.ibexa-list-filters__item--type');
+    const datetimeFilterNodes = filtersContainerNode.querySelectorAll('.ibexa-list-filters__item--date-time .ibexa-picker');
+
+    const clearFilter = (filterNode) => {
+        if (!filterNode) {
+            return;
+        }
+
+        const sourceSelect = filterNode.querySelector('.ibexa-list-filters__item-content .ibexa-dropdown__source .ibexa-input--select');
+        const checkboxes = filterNode.querySelectorAll(
+            '.ibexa-list-filters__item-content .ibexa-input--checkbox:not([name="dropdown-checkbox"])',
+        );
+        const timePicker = filterNode.querySelector('.ibexa-date-time-picker__input');
+
+        if (sourceSelect) {
+            const sourceSelectOptions = sourceSelect.querySelectorAll('option');
+            sourceSelectOptions.forEach((option) => {
+                option.selected = false;
+            });
+
+            if (isTimeFilterNode(filterNode)) {
+                sourceSelectOptions[0].selected = true;
+            }
+        } else if (checkboxes.length) {
+            checkboxes.forEach((checkbox) => (checkbox.checked = false));
+        } else if (timePicker.value.length) {
+            const formInput = filterNode.querySelector('.ibexa-picker__form-input');
+
+            timePicker.value = '';
+            formInput.value = '';
+
+            timePicker.dispatchEvent(new Event('input'));
+            formInput.dispatchEvent(new Event('input'));
+        }
+
+        searchForm.submit();
+    };
+    const attachStatusFilterEvents = (filterNode) => {
+        if (!filterNode) {
+            return;
+        }
+
+        const checkboxes = filterNode.querySelectorAll(
+            '.ibexa-list-filters__item-content .ibexa-input--checkbox:not([name="dropdown-checkbox"])',
+        );
+        checkboxes.forEach((checkbox) => {
+            checkbox.addEventListener('change', filterChange, false);
+        });
+    };
+    const attachTypeFilterEvents = (filterNode) => {
+        if (!filterNode) {
+            return;
+        }
+
+        const sourceSelect = filterNode.querySelector('.ibexa-list-filters__item-content .ibexa-dropdown__source .ibexa-input--select');
+        sourceSelect?.addEventListener('change', filterChange, false);
+    };
+    const attachDateFilterEvents = (filterNode) => {
+        if (!filterNode) {
+            return;
+        }
+
+        const picker = filterNode.querySelector('.ibexa-input--date');
+        picker?.addEventListener('change', dateFilterChange, false);
+    };
+
+    const isTimeFilterNode = (filterNode) => {
```

Comment:
strange name, personally I have no idea what it represents and what I should expect from it... is node date/time picker?